### ISSUE TITLE
Error when installing with JHipster 4.0.1

### ISSUE THREAD

**Issue by mreddimasi** [hermeslm/generator-jhipster-angular-datatables#3]
Hi,
I tried installing this module on a Jhipster 4.0.1 application but get the following error

```
yo jhipster-angular-datatables default
Composing JHipster configuration with module angular-datatables
Welcome to the JHipster-Angular-DataTables Generator! v1.3.4

This will change JHipster Entity list component with Angular-DataTables

Error jhipster-angular-datatables default

ERROR! I support only JHipster versions greater than ${supportedJHVersion}... If you want to use Entity Audit with an older JHipster version, download a previous version that supports the required JHipster version.
```
Would it not work with Jhipster 4.0.1 version? This is on Windows 10 OS

Thank You

**Comment by Ledar** [comment]
I meet the same error, and i install the jhipster with yarn , but not npm

**Comment by hermeslm** [comment]
This module is not compatible with this version of Jhipster 4, so I'm so sorry about this, I need to check and fix this compatibility issue but I'm very busy, if you want to contribute any help will be appreciate, I will try to resolve this issue coming soon.

**Comment by Ledar** [comment]
Thanks, and how can i revert my jhipster version to 3?

**Comment by hermeslm** [comment]
I Fixed all error and tested with JHipster v4.0.3 too:

1)  Fixed erros with Jhipster version checker.
yo jhipster-angular-datatables default
Composing JHipster configuration with module angular-datatables
Welcome to the JHipster-Angular-DataTables Generator! v1.3.4

This will change JHipster Entity list component with Angular-DataTables

Error jhipster-angular-datatables default

ERROR! I support only JHipster versions greater than ${supportedJHVersion}... If you want to use Entity Audit with an older JHipster version, download a previous version that supports the required JHipster version.

2) Fixed error with default param in the generator.
